### ISSUE TITLE
School Space

### ISSUE THREAD

**Issue by nickevansuk** [openactive/activation#240]
[//]: # (Simply update the name of this issue above to the name of your system, then click the green "Submit new issue" button below)

## Development Progress Tracker

This issue aims to provide public visibility of this new OpenActive implementation's development progress. The task list below tracks high level progress at-a-glance, while comments on this issue are used to provide updates in more detail.

### Speculative
- [x] OpenActive community engagement
- [x] Scoping
- [x] Scheduling

### In Development
- [x] Dataset site built
- [x] Dataset site GitHub issues board created
- [x] Open data feeds built
- [x] Open data feeds pass OpenActive Validator
- [x] Open data feeds pass OpenActive Test Suite
- [x] Open Booking API built
- [x] OpenActive Conformance Certificate generated (via OpenActive Test Suite)

### In Review
- [x] OpenActive community feedback
- [ ] Deployed to production
- [ ] First integration live

### Certified and Live
- [ ] Added to OpenActive Data Catalog

**Comment by nickevansuk** [comment]
This is blocked on School Space's interest in accepting ad-hoc bookings (they currently only work with block bookings)
